app/src/components/messageParser.js

```diff
@@ -48,6 +51,12 @@ const messageParser = {
       env: (obj && obj.env) ? obj.env : 'dev'
     };
 
+    if (obj && obj.transaction && obj.transaction === Object(obj.transaction)) {
```

Comment:
I'm sure it works but what's this test for `obj.transaction === Object(obj.transaction)`?

Reply:
to make sure that obj.transaction is an Object.

Reply:
Huh, so it does. I always did typeof and a null check, but googling around that's done as well. Carry on :)